kong/templates/nginx_kong.lua

```diff
@@ -168,6 +168,11 @@ server {
         proxy_set_header      X-Forwarded-Path   $upstream_x_forwarded_path;
         proxy_set_header      X-Forwarded-Prefix $upstream_x_forwarded_prefix;
         proxy_set_header      X-Real-IP          $remote_addr;
+> for _, name in ipairs(headers_upstream) do
+>   if name == "x-kong-request-id" then
+        proxy_set_header      X-Kong-Request-Id  $kong_request_id;
```

Comment:
Could we access `kong.configuration.enabled_headers_upstream`?
Or else we should add another line `> break`.

Reply:
modified to `> break` and `lower()`, but need some time to examine whether we can access `kong.configuration.enabled_headers_upstream` here

Reply:
I haved test if we can access the field `kong`, its nil. and also cannot access `enabled_headers_upstream` directly here.

Reply:
also note that i use `headers_upstream` . I copied this style from other part of this template.lua. :)